### DISCUSSION TITLE
UI redesign to complete material m3 design, show subreddit(s) on the search results page

### DISCUSSION THREAD

**Discussion by idontveapc** [Docile-Alligator/Infinity-For-Reddit#1407] [Ideas]
focus for reddit feels similar to infinity but it has good material you look, not so working though

### YOU MUST MAKE BETTER THAN HIM.

**Comment by idontveapc** [comment]
is focus for reddit copying you or you copied them?

**Comment by us3rname101** [comment]
yes docile-alligator definetly copied them, focus was published on 28.3.2023 and infinity has been in deveplopment for years.

**Comment by us3rname101** [comment]
Seems like a very cursed app though. Paywalled features that Infinity has for free and has ads. Only good thing about it that it has somewhat a m3 design.

**Reply by us3rname101** [reply]
I think the person is breaking the license though. "When a modified version is used to provide a service over a network, the complete source code of the modified version must be made available." modified version = Focus, a service over a network = reddit, must be made available = is not

**Comment by idontveapc** [comment]
> Only good thing about it that it has somewhat a m3 design.

yup!
